openstack/sharedfilesystems/v2/shares/requests.go

```diff
@@ -79,6 +79,7 @@ func Get(client *gophercloud.ServiceClient, id string) (r GetResult) {
 }
 
 // GetExportLocations will get shareID's export locations
+// client must have Microversion set; minimum supported microversion for Get Export Locations is 2.14
```

Comment:
We'll need some punctuation here. The full comment should now read:

```
GetExportLocations will get shareID's export locations. Client must have 
Microversion set; minimum supported microversion for GetExportLocations 
is 2.14.
```

Reply:
Done in next change.